engine/crates/engine-config-builder/src/from_sdl_config.rs

```diff
@@ -211,7 +211,7 @@ impl<'a> BuildContext<'a> {
                     retry,
                     entity_caching: entity_caching.as_ref().map(|config| match config {
                         EntityCachingConfig::Disabled => EntityCaching::Disabled,
-                        EntityCachingConfig::Enabled { ttl } => EntityCaching::Enabled { ttl: *ttl },
+                        EntityCachingConfig::Enabled { ttl, .. } => EntityCaching::Enabled { ttl: *ttl },
```

Comment:
:zap: **Suggestion** 
Ensure that the handling of additional fields in the pattern matching is consistent across all usages to avoid potential bugs.